metadata-load/src/main/resources/application.properties

```diff
@@ -8,3 +8,23 @@ ena.datasource.tomcat.maxActive=2
 
 # Analysis SQL query
 ena.analysis.query=SELECT ANALYSIS_XML FROM ERA.ANALYSIS WHERE ANALYSIS_ID = :accession
+
+
+spring.application.name=metadata-import
+
+# Spring boot uses a generic domain when loading in a Tomcat environment. When multiple spring boot applications run
+# under the same tomcat this makes that applications try to register its beans and clash with beans already defined
+# by other spring-boot application. This usually break the application initialization. By setting a jmx domain for
+# this application beans we avoid this clash.
+endpoints.jmx.domain=metadata-import
+spring.jmx.default-domain=metadata-import
+
+endpoints.jmx.unique-names=true
+
+# Metadata Database connection details
```

Comment:
Move these properties closer to the ENA database ones, to try to keep the structure of the properties file a bit more cohesive.

Reply:
I am now wondering, why not using `spring.datasource` properties to get the benefits of the autoconfiguration for that data source?